src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -326,6 +474,14 @@ protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
             }
         }
 
+        private static BoundNode Rewrite(
```

Comment:
>Rewrite [](start = 33, length = 7)

I think this method is used in a single place (in `AnalyzeAndRewrite`). Consider inlining. Otherwise, it feels like it should be a static method on `NullabilityRewriter`.